esx_service/utils/auth_api.py

```diff
@@ -457,7 +457,7 @@ def _tenant_create(name, default_datastore, description="", vm_list=None, privil
 
     # VM list can be empty during tenant create. Validate only if it exists
     vms = None
-    if vm_list:
+    if vm_list and name != auth_data_const.DEFAULT_TENANT:
```

Comment:
Adding a VM to the default group is already handled,
/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup vm add --name _DEFAULT --vm-list master-VM0.0
This feature is not supported for vmgroup _DEFAULT.


I wasn't sure we wanted to error out and have the user redo the command, may be issue a warning and let the command continue.